cachi2/core/package_managers/npm.py

```diff
@@ -502,6 +561,95 @@ def _get_npm_dependencies(
     return download_paths
 
 
+def replace_dependency(dependency_version: str) -> bool:
```

Comment:
nitpick on the naming, this is more of a `should_replace_dependency`

I would expect a function called `replace_dependency` to replace a dependency :grin:

Reply:
True, fixed